project/driver_controls/inc/mechanical_brake_state.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+#include "fsm_state.h"
```

Comment:
Why is `fsm_state.h` included in each FSM's header? Don't you only need `fsm.h`?